django/contrib/postgres/constraints.py

```diff
@@ -0,0 +1,90 @@
+from django.db.models import CheckConstraint, F
+from django.db.models.sql import Query
+
+__all__ = ['ExcludeConstraint']
+
+
+class ExcludeConstraint(CheckConstraint):
```

Comment:
Might want to extend `BaseConstraint` instead? Also it seems `ExclusionConstraint` would be more appropriate?

Reply:
Agreeing to both suggestions :+1: